### PR TITLE
First commit of the Fivetran Logs package

### PR DESCRIPTION
**Pull request by dwl285**
Fivetran logs is a free connector fivetran offers, sending a log of all data related to fivetran data ingestions to your warehouse. This package models that data into something useful for analytics and reporting.

A few things to note:
 - only supports BigQuery for now
 - only supports looking for the data in the defaultDatabase
 - no support for Fivetran transformations data (because we don't have any, so I can't test it!)
 - only supports a single sync destination warehouse

### CODE REVIEW

File: definitions/example.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 // TODO: update package name
-const package = require("../");
+const fivetran_logs = require("../");
 
-const models = package({
-  // TODO: add variables
+const models = fivetran_logs({
+  // TODO: rename to fivetranLogSchema
```
Comment (lewish) [inline definitions/example.js]: Looks like you can remove

File: includes/connector_daily_api_calls.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  connector.destination_name,
+  connector.destination_id,
+  connector.set_up_at
+from
+  ${ctx.ref(params.defaultConfig.schema, "fivetran_log_connector_status")} as connector
+  left join api_calls on api_calls.connector_name = connector.connector_name
+),
+
+/* TODO: Make this not just BigQuery specific */
+dates as (
```
Comment (lewish) [inline includes/connector_daily_api_calls.js]: Why bother with this? Do we really need to create a row for every date, why not just leave as nulls? Dramatically simplifies the queries below and a lot of the BQ specific stuff, and removes the need for a first date param.

File: includes/connector_status.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+where
+  event_type = 'SEVERE'
+  or event_type = 'WARNING'
+  or event_subtype like 'sync%'
+),
+
+schema_changes as (
+select
+  connector_name,
+  count(*) as number_of_schema_changes_last_month
```
Comment (lewish) [inline includes/connector_status.js]: nit: not entirely correct

File: index.js [hunk 4]
```diff
@@ -1,21 +1,101 @@
...
+    ...params
+  };
 
-    // Publish and return datasets.
-    let result = {
+  let account, account_membership, active_volume, connector, credits_used, destination, destination_membership, log, user;
+
+  account = declare({
+    ...params.defaultConfig,
+    schema: params.fivetranLogSchema,
```
Comment (lewish) [inline index.js:29]: I would consider adding support for overriding the `fivetranLogDatabase` too, as I suspect it's not uncommon to have these logs go into a different project/database than their default dataform one!

**Review by lewish — APPROVED** [review]
Started reading the SQL but there was so much, and sounds like this is coming from elsewhere anyway so stopped. Overall structure looks good to me!

File: index.js (hunk 4 above)
Comment (dwl285) [reply index.js:29]: I'm going to save that for a future update, but I do agree

**Pull request merged**